src/compose/images.ts

```diff
@@ -196,6 +196,14 @@ export async function triggerFetch(
 		});
 	};
 
+	// Remove image task on fetch abort to prevent noop loop
```

Comment:
I don't understand the comment. Why would this cause a noop loop?

Reply:
If the AbortSignal is triggered, ImageTask's `onFinish` wasn't being called, so the state engine assumed that the ImageTask was still running, and would generate `noop` steps while waiting for the image to finish downloading. In reality though, the image was no longer being downloaded, so the state engine would wait forever.